### PR TITLE
Implemented the PrintMenuProgramBanList method. Closed #31

### LINKED ISSUE #31: Implement the PrintMenuProgramBanList method
Implement the PrintMenuProgramBanList method in the ConsoleUserInterface class. Add sub-menu items to add and remove from the list of banned words.

### PR DESCRIPTION
**Pull request by xs887z**

### CODE REVIEW

**Review by TopAcademy2022 — APPROVED** [review]

**Review by cany8042 — APPROVED** [review]

**Review by Mi-stik31 — APPROVED** [review]

**Review by SugarSalt9 — APPROVED** [review]

**Review by Andrey31ts — APPROVED** [review]

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 1]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+            }
+        }
+
+        private void AddProgramToBanList()
+        {
+            Console.WriteLine("Введите название программы для добавления в список");
+            string programName = Console.ReadLine();
+            _programbanlist.AddProgram(programName);
+            Console.WriteLine($"Программа '{programName}' добавлена в список запрещённых");
+            Console.WriteLine("\nНажмите клавишу чтобы продолжить");
```
Comment (London3149-train) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:266]: Remove '\n'.

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 2]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+            Console.ReadKey(true);
+        }
+
+        private void RemoveProgramFromBanList()
+        {
+            Console.WriteLine("Введите название программы для удаления из списка");
+            string programName = Console.ReadLine();
+            _programbanlist.RemoveProgram(programName);
+            Console.WriteLine($"Программа '{programName}' удаленна из списка запрещённых программ");
+            Console.WriteLine("\nНажмите клавишу чтобы продолжить");
```
Comment (London3149-train) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:276]: Remove '\n'.

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 3]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+        {
+            Console.WriteLine("Список запрещённых программ:");
+            List<string> banList = _programbanlist.GetProgramBanList();
+            if (banList.Count == 0)
+            {
+                Console.WriteLine("\nСписок запрещенных программ пуст.\n");
+            }
+            else
+            {
+                Console.WriteLine("\nСписок запрещенных программ:");
```
Comment (London3149-train) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:290]: Remove line.

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 4]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+
+                foreach (string Program in banList)
+                {
+                    Console.WriteLine(Program);
```
Comment (London3149-train) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:294]: Edit method to Console.Write and add space inter-polis. Example Console.Write($"Program ").

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 5]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+                }
+                Console.WriteLine("\nНажмите клавишу чтобы продолжить");
```
Comment (London3149-train) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:296]: Remove '\n'.

**Review by London3149-train — CHANGES_REQUESTED** [review]

File: user-monitoring-gui/Services/ConsoleUserInterface.cs [hunk 6]
```diff
@@ -211,8 +213,91 @@ public void PrintBanList()
...
+            List<string> banList = _programbanlist.GetProgramBanList();
+            if (banList.Count == 0)
+            {
+                Console.WriteLine("\nСписок запрещенных программ пуст.\n");
+            }
+            else
+            {
+                Console.WriteLine("\nСписок запрещенных программ:");
+
+                foreach (string Program in banList)
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:292]: Rename 'Program' variable, because this class name.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**